File: Servidor/InHomeTFGCarlosHernandez/src/main/java/org/tfg/inhometfgcarloshernandez/spring/model/request/CrearEventoRequestDTO.java
package org.tfg.inhometfgcarloshernandez.spring.model.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.tfg.inhometfgcarloshernandez.spring.model.EventoDTO;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class CrearEventoRequestDTO {
    private int idCasa;
    private EventoDTO eventoCasa;
    private String fecha;
}

File: Cliente/InHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/inhometfgandroidcarloshernandez/data/model/request/CrearEventoRequestDTO.kt
package com.example.inhometfgandroidcarloshernandez.data.model.request

import com.example.inhometfgandroidcarloshernandez.ui.framework.screens.calendario.CalendarioContract

data class CrearEventoRequestDTO (
    var idCasa: String,
    var eventoCasa: CalendarioContract.EventoCasa,
    var fecha:String
)